### DISCUSSION TITLE
how do i get the control of the prompt back if i add a seq_add_callback with an error

### DISCUSSION THREAD

**Discussion by ely-peddler** [shorepine/tulipcc#391] [General]
calling seq_add_callback and passing a function with errors sends the prompt into an infinite loop, spitting out error messages, can it be  recovered without rebooting?

**Comment by bwhitman** [comment]
Nothing easy. The sequencer is always running in tulip and can't be stopped. You could blindly type tulip.seq_remove_callback(slot) or redefine your broken callback with a do nothing one , like def cb: return False or etc.  or just reboot with the reset button or control-D.
